style.css

```diff
@@ -0,0 +1,7 @@
+main {
+  background-color: gray;
+}
+
+h1 {
+  color: orange;
+}
```

Comment:
- [ ] It is better to use hex codes or RGB values for colors instead of using the original names of them.